scripts/qiita-test-install

```diff
@@ -238,7 +238,7 @@ class QiitaConfig(TestCase):
         if not self.psql_version:
             self.assertTrue(False, 'PostgreSQL not running or configured')
 
-        acceptable_version = (9, 1, 14)
+        acceptable_version = (9, 3, 0)
```

Comment:
I changed it for the postgres case. However I think the wording is incorrect for all of them as all of them are checking using `<=` rather than `==`, meaning that is a minimum required version.